src/js/extension-anchor.js

```diff
@@ -24,6 +24,10 @@ var AnchorExtension;
 
         // Button and Extension handling
 
+        // labels for the anchor-edit form buttons
+        formSaveLabel: '&#10003;',
+        formCloseLabel: '&times;',
```

Comment:
they actually still do use the buttonLabels===fontawesome path, I did this this way to prevent potentially regressing anyone. these two values were simply fallback/defaults in the template rendering, so I exposed them so I could override personally in a non-fontawesome path. if you are setting buttonLabels: "fontawesome" you likely have never seen the `&times;` anyway ... so to answer your question, the fa-labels are preserved by virtue of this just being extracting of a fixed value into a property.  I had thought about moving the buttonlabels===fa check into init and building up formSaveLabel/formCloseLabel one time, but the complicated things by needing to check for a value prior, and yadda yadda yadda ...